src/http/RequestBuilder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace markhuot\craftpest\http;
+
+use craft\web\User;
+use markhuot\craftpest\http\requests\GetRequest;
+use markhuot\craftpest\http\requests\PostRequest;
+use markhuot\craftpest\http\requests\WebRequest;
+use markhuot\craftpest\web\Application;
+use markhuot\craftpest\web\TestableResponse;
+use yii\web\Cookie;
+
+class RequestBuilder
+{
+    private WebRequest $request;
+    private Application $app;
```

Comment:
If no `$app` is passed in to the constructor we set this to `\Craft::$app` (down on line 25). That defaults to a `\craft\web\Application` not the currently typed `markhuot\craftpest\web\Application`. I haven't looked in to if we can override this but I'd be hesitant to reach that far in to Craft for typing. Not sure _what_ to do here, though…